app/models/audit.rb

```diff
@@ -41,6 +41,14 @@ class Audit < ApplicationRecord
   scope :published, -> { where(published: true) }
   scope :by_date,   -> { order(created_at: :desc) }
 
+  def completed_activity_types
+    activity_types.where(id: school.activities.where(happened_on: created_at..).pluck(:activity_type_id))
```

Comment:
I didn't know about this syntax!

Reply:
It's fairly new to me too, only found it a few weeks ago! I quite like it.